tsinfer/formats.py

```diff
@@ -2512,10 +2512,21 @@ def individuals_metadata_schema(self):
     @functools.cached_property
     def individuals_metadata(self):
         schema = tskit.MetadataSchema(self.populations_metadata_schema)
-        try:
-            return [schema.decode_row(r) for r in self.data["individuals_metadata"][:]]
-        except KeyError:
-            return [{} for _ in range(self.num_individuals)]
+        if "individuals_metadata" in self.data:
```

Comment:
Took me a few minutes to figure this out - maybe a comment explaining that we always include the sgkit_sample_id, whether or not the user specified other metadata? What happens if the user already has ``sgkit_sample_id`` defined in their metadata?

This needs a CHANGELOG update too.

Reply:
I think if the key already exists that we silently don't set it. I don't think it is worth a warning as sgkit_sample_id is so specific it is likely to be the same string.

Do we need a changelog? None of the sgkit stuff is released yet.

Reply:
True enough - I can see us forgetting to document this addition to the metadata though, so worth starting the CHANGELOG for sgkit stuff now?